src/main/java/org/openjdk/shenandoah/RegionStat.java

```diff
@@ -73,6 +73,19 @@ public RegionStat(float usedLvl, float liveLvl, float tlabLvl, float gclabLvl, f
         this.affiliation = RegionAffiliation.YOUNG;
         this.showLivenessDetail = Boolean.getBoolean("show.liveness");
     }
+    //This constructor is for CounterTest
+    public RegionStat(float usedLvl, float liveLvl, float tlabLvl, float gclabLvl, float plabLvl, float sharedLvl, RegionAffiliation affiliation,RegionState state) {
+        this.usedLvl = usedLvl;
+        this.liveLvl = liveLvl;
+        this.tlabLvl = tlabLvl;
+        this.gclabLvl = gclabLvl;
+        this.plabLvl = plabLvl;
+        this.sharedLvl = sharedLvl;
```

Comment:
Noob question: what is "Shared" here? I am sure it's some Shenandoah nomenclature, but wondered what it means. Thanks!

Reply:
Shared means shared allocations, which basically means Shenandoah allocated objects outside of a TLAB. This is the explanation that William gave me.

Reply:
ok. We've traditionally used the term "direct allocation", but "shared" makes sense since it's out of shared (rather than already allocated to thread). Thanks for the help with the term!